bot/mozdata.ini

```diff
@@ -0,0 +1,2 @@
+[User-Agent]
+name = code-review-bot/1.8.4
```

Comment:
We'll have to remember to update this when we release new versions. Maybe we could add a placeholder here and replace it with a command in Dockerfile?

WDYT @La0 ?

Reply:
It would actually be simpler to implement our own `Config` class that simply provides the user-agent value, as that's the only one we want to provide.

I'll make a small PR to show this, then you can chosse

Reply:
I made #2452 to use a Local configuration